FORMAT_ADIOS.md

```diff
@@ -32,3 +32,28 @@ Output from `bpls -A` for a boolean attribute `pybool` stored in the location of
 
 There is no convention yet for a unique representation of ADIOS2 variables with boolean type.
 Thus, implementations should cast the data to and from `unsigned char` instead.
+
+## `stepBased` Encoding of Iterations
```

Comment:
Rename: I am not sure why, but for some reason we now call this `variableBased` in https://github.com/openPMD/openPMD-api/pull/855

@franzpoeschel let's clarify what we pick, shall I update the standard PR to be named `variableBased`, too?

Reply:
Yep, we discussed this a while ago and came to the conclusion to call it variable-based since steps are an ADIOS2-specific feature, but this encoding generally relies on a backend's ability to have *variable* datasets.